FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -388,14 +376,14 @@ class ModelInfoRetriever {
             }
           }
         }
-      /// FIS token error.
+      /// Error retrieving auth token.
```

Comment:
Nit: 2 `//` only

Reply:
Updated everywhere not a property/method/class description to be `//`.